game-core/src/main/resources/META-INF/triplea/game_engine.properties

```diff
@@ -0,0 +1 @@
+engine_version = 1.9.0.0.dev
```

Comment:
I have mixed feelings on this.
It's technically not a property anymore, so in theory we'd have just been fine with simply putting the version string without anything else in this?
I know that it's easier to implement because the structure is already there to parse files in the properties format, but on the same time we're not really expecting any property to be added to this file.
Also this isn't really game_engine.properties anymore, `version.properties` (if it remains a properties file) would be clearer IMO

Reply:
Agreed, and that's where I was going when I decided this PR was "just the right size." :smile:  I plan on following this up with another PR that restructures and renames this particular aspect a little better.